rfc6962-bis.xml

```diff
@@ -1468,10 +1465,10 @@ but it is expected there will be a variety.
           <t>
             <list style="symbols">
               <t>
-                In the case that use of TLS with a valid certificate is mandated by explicit security policy, application protocol specification, or other means, the TLS client MUST refuse the connection.
+                In the case that use of TLS with a valid certificate is mandated by explicit security policy, application protocol specification, or other means, the TLS client SHOULD refuse the connection.
```

Comment:
We liked https://tools.ietf.org/html/rfc7633#section-4.3.3 so we copied'n'pasted it.  :-)

It's a conditional MUST, not an absolute MUST.  When the "In the case that...mandated by..." condition is met, MUST is appropriate.  (IIRC we wanted to avoid absolute MUSTs/SHOULDs, because specifying client behaviour is out of scope for 6962-bis).

How about we change the 2 occurrences of "(the) use of TLS with a valid certificate" to "compliance" ?